docs/1.6_lucra_event_listener.md

```diff
@@ -0,0 +1,43 @@
+# Lucra Event Listener
+
+### Registering `onMessage` functions
+
+If you need to register a handler function for any of the `onMessage` callbacks, you can do so after instantiation like so:
+
+```typescript
+const lucraClient = new LucraClient(...)
+lucraClient.userInfoHandler = handleUserInfo;
+```
+
+### Messages from LucraClient
+
+- `loginSuccess` - the user has logged into Lucra
+- `userInfo` - whenever an update happens to the user, the callback to this function will receive the newest version of that user
+- `matchupCreated` - the user successfully created a matchup, and contains the id of that matchup
+- `matchupStarted` - the matchup owner started the matchup, and contains the id of that matchup
+- `matchupAccepted` - the user successfully joined someone else's matchup, and contains the id of that matchup
+- `matchupCanceled` - the user successfully canceled the matchup, and contains the id of that matchup
+- `matchupInviteUrl` - Lucra is requesting an invite URL for a specific matchup. Use `matchupDeepLinkHandler` to handle this. Your handler receives the `matchupId` and must call `sendMessage.deepLinkResponse()` with the generated URL.
+- `convertToCredit` - if Convert to Credit is enabled, Lucra will call this method with the desired amount to convert to credit. Respond with a `convertToCreditResponse` message defined below
+- `deepLink` - Lucra is requesting a url that the user of the SDK will open to then open up the LucraClient at where the deepLink url is linking to. For matchup invitations, use `matchupDeepLinkHandler`.
+- `tournamentJoined` - the user successfully joined a tournament, and contains the id of the tournament
+- `navigationEvent` - the user has navigated somewhere in Lucra, contains the full url that you can use later when doing `.open().deepLink(url)`
+- `claimReward` - the user is redeeming a Free To Play reward
+
+### Messages you can send to LucraClient
+
+`userUpdated` - update the LucraClient user with known information about the current user. This information will be persisted to LucraSports and will be used to pre-fill forms the user will have to fill out for things like Identity Verification
+
+You may optionally include a `metadata` property as valid JSON related to the Lucra user. Key/value pairs should be of type `String` only. _Note_: On the `metadata` object, the `external_id` property will be used to index, if provided. Passing in metadata in this way will be used to link client users with the Lucra user for autosettlement identification purposes.
+
+### Sending a message to LucraClient
+
+```
+const lucraClient = new LucraClient(...)
+lucraClient.sendMessage.userUpdated(...)
+lucraClient.sendMessage.convertToCreditResponse(...)
+lucraClient.sendMessage.enableConvertToCredit()
+lucraClient.sendMessage.deepLinkResponse(...)
+lucraClient.sendMessage.availableRewards(...)
```

Comment:
Can you break these down into individual subheaders with the description of what is does and example payloads? These are usually the most complicated part of our SDK. User Configuration, convert to credit and rewards probably deserve their own pages. Just like deeplinks